monitoring/lifecycle/alerts.test.yaml

```diff
@@ -611,3 +611,25 @@ tests:
       - alertname: KafkaConsumerRebalanceTimeouts
         eval_time: 4m
         exp_alerts: []
+
+  - name: CircuitBreakerError
+    interval: 1m
+    input_series:
+      - series: s3_circuit_breaker_err{namespace="zenko"}
+        values: 0 0 1 1 1
+    alert_rule_test:
+      - alertname: CircuitBreakerError
+        eval_time: 0m
+        exp_alerts: []
+      - alertname: CircuitBreakerError
+        eval_time: 5m
+        exp_alerts:
+          - exp_labels:
+              severity: warning
+              namespace: zenko
+            exp_annotations:
+              summary: "Circuit breaker is unable to query prometheus"
+              description: "Circuit breaker is unable to query prometheus."
+      - alertname: CircuitBreakerError
+        eval_time: 10m
```

Comment:
the time series in this test does not reach 10m : so you are not actually testing that the alert stops when things go back to normal, but instead testing that the alert disappear when there is no an (s3_circuit_breaker_err) metric anymore...

Reply:
The test has been adapted to check the 10m interval https://github.com/scality/backbeat/pull/2516/commits/32be5a866e3f8db3da1d9222236041b735ef32e8#diff-08e5453052d3ff64a7ec5f78d40faf7811de2787e942a284b0263f118de6a13aR619